scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -221,11 +227,15 @@ elif [ "$os" == "centos7" ] || [ "$os" == "centos8" ] ||
     elif [ "$os" == "centos8" ] || [ "$os" == "rhel8" ] || [ "$os" == "almalinux8.7" ] ; then
         pkgs_core+=(
             https://vault.centos.org/centos/8/AppStream/$arch/os/Packages/tbb-2018.2-9.el8.$arch.rpm
+            https://vault.centos.org/centos/8/AppStream/$arch/os/Packages/ocl-icd-2.2.12-1.el8.$arch.rpm
```

Comment:
ocl-icd is a part of NEO installation script

Reply:
fixed